cdist/conf/type/__acl/man.rst

```diff
@@ -21,6 +21,9 @@ user
 group
    Add group ACL entry.
 
+mask
+   Add mask ACL entry.
```

Comment:
this should be in "Optional" section :smiling_imp:

Reply:
Fixed, amended the 1st commit.